packages/reports/src/Stores/Modules/Statement/Helpers/format-response.ts

```diff
@@ -1,36 +1,42 @@
 import { formatMoney, toTitleCase, toMoment, getMarketInformation, getSymbolDisplayName } from '@deriv/shared';
 import { localize } from '@deriv/translations';
+import { ActiveSymbols, Statement } from '@deriv/api-types';
 
-export const formatStatementTransaction = (transaction, currency, active_symbols = []) => {
+export const formatStatementTransaction = (
```

Comment:
I've migrated this function to TS in order to fix types of MarketSymbolIconRow component props. It's necessary for proper testing.